frontend/packages/yki/src/tests/cypress/support/page-objects/publicRegistrationPage.ts

```diff
@@ -2,9 +2,18 @@ import { selectComboBoxOptionByName } from 'tests/cypress/support/utils/comboBox
 
 class PublicRegistrationPage {
   elements = {
-    filterByLanguage: () =>
-      cy.findByRole('combobox', { name: /Valitse kieli/ }),
-    filterByLevel: () => cy.findByRole('combobox', { name: /Valitse taso/ }),
+    filterByLanguage: (isPhone: boolean = false) =>
+      isPhone
+        ? cy.findAllByRole('combobox').eq(0).should('be.visible')
+        : cy.findByRole('combobox', { name: /Valitse kieli/ }),
+    filterByLevel: (isPhone: boolean = false) =>
+      isPhone
+        ? cy
+            .findAllByRole('combobox')
+            .eq(1)
+            .should('be.visible')
+            .should('be.visible')
```

Comment:
Onko tällä `.should('be.visible')` toistolla jokin tarkoitus vai onko vain jäänyt epähuomiossa?